packages/react-ui/src/theme/themeUtils.js

```diff
@@ -2,3 +2,11 @@ import createSpacing from '@material-ui/core/styles/createSpacing';
 import { SPACING } from './themeConstants';
 
 export const spacing = createSpacing(SPACING);
+
+// Convert pixels to rem
+export function getPixelToRem(px) {
```

Comment:
There should be probably a function in mui for this, but I'm ok with having our own, Probably we could switch in the future, if required

Reply:
I didn't find anyone, but maybe you're right. I'll look into it again after the migration.